src/dhcp_fsm.erl

```diff
@@ -198,7 +198,7 @@ bound(Pkg = #dhcp_package{xid = _XId, message_type = request},
             case delegate(request, Pkg, State) of
                 {ok, RPkg, State1} ->
                     Timeout = dhcp_package:get_option(ip_address_lease_time, RPkg),
-                    {next_state, bound, State#state{last=erlang:now(),
+                    {next_state, bound, State#state{last=current_time(),
                                                     handler_state = State1}, Timeout};
```

Comment:
Line 202 is too long:                                                     handler_state = State1}, Timeout};.